components/content_settings/renderer/brave_content_settings_agent_impl.cc

```diff
@@ -332,6 +332,8 @@ void BraveContentSettingsAgentImpl::DidCommitProvisionalLoad(
 
 BraveFarblingLevel BraveContentSettingsAgentImpl::GetBraveFarblingLevel(
     ContentSettingsType webcompat_settings_type) {
+  GetOrCreateBraveShieldsRemote()->OnWebcompatFeatureInvoked(
```

Comment:
`Invoked` -> `Changed/Modified`?

Reply:
"Invoked" here refers to whether the Web API (or similar fingerprintable feature) has been invoked by the web page.